_posts/2016-03-17-the-need-for-fit.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "The need for fit"
+date:   2016-03-17 11:58:26
+author: Michael Peyton Jones
+categories:
+---
+
+Personal fit with project the is incredibly important in startup founders. 
+Startup work is demanding: it typically involves working long hours for several years,
+with the prospect of a high failure rate even if you do keep going.
+This can be understandably [demoralizing](http://www.paulgraham.com/die.html),
+so a good personal fit with the project improves your chances of success both by
+making you more effective at the object level work, and by preventing failure
```

Comment:
I think we need to define personal fit here. It could be holden's wide definition that seems almost like 'type' or 'role' or it could be 80k's narrower definition as 'chance of excelling at the job'